docs/content/en/docs/implementing/restart-application-deployment/_index.md

```diff
@@ -3,32 +3,34 @@ title: Restart an Application Deployment
 description: Learn how to restart an unsuccessful Keptn Application Deployment.
 icon: concepts
 layout: quickstart
-weight: 20
-hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
+weight: 100
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
-## Restart an Application Deployment
-
-During the deployment of a `KeptnApp`, it might be that the deployment fails due to an unsuccessful pre-deployment
-evaluation or pre-deployment task.
-This could happen because of, e.g., a misconfigured target value of a `KeptnEvaluationDefinition`, or a wrong URL being
-checked in a pre deployment check.
-
-To retry a `KeptnApp` deployment without incrementing the version of the `KeptnApp`, we introduced the concept of **
-revisions** for a `KeptnAppVersion`.
-This means that
-whenever the spec of a `KeptnApp` changes, even though the version stays the same, the KLT Operator will create a new
-revision of the `KeptnAppVersion` referring to the `KeptnApp`.
-
-This way, when a `KeptnApp` failed due to a misconfigured pre-deployment check, you can first fix the configuration of
-the `KeptnTaskDefinition`/`KeptnEvaluationDefinition`, then
-increase the value of `spec.revision` of the `KeptnApp` and finally apply the updated `KeptnApp` manifest.
-This will result in a restart of the `KeptnApp`.
-Afterwards, all related `KeptnWorkloadInstances` will automatically refer to the newly
-created revision of the `KeptnAppVersion` to determine whether they are allowed to enter their respective deployment
-phase.
-
-To illustrate this, let's have a look at the following example:
+A [KeptnApp](../../yaml-crd-ref/app.md) can fail
+because of an unsuccessful pre-deployment evaluation
+or pre-deployment task.
+For example, this happens if the target value of a
+[KeptnEvaluationDefinition](../../yaml-crd-ref/evaluationdefinition.md)
+resource is misconfigured
+or a pre-deployment evaluation checks the wrong URL.
+
+Increment the `spec.revision` field of `KeptnApp`
+to retry a `KeptnApp` deployment
+without incrementing the version of the `KeptnApp`
```

Comment:
I think this is a left-over. The paragraph after these lines covers this text in more detail :)